client/app/stories/index.jsx

```diff
@@ -154,3 +155,18 @@ storiesOf('Footer', module)
   .add('View', withInfo({})(() =>
     <Footer />,
   ));
+
+storiesOf('Avatar', module)
+  .add('With name', withInfo({})(() =>
+    (<Avatar
+      src="https://s3-alpha.figma.com/img/6c65/47d3/875d99c169116da7bcaa7cefb0dc2ece?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAJK6APQGEHTP6I3PA%2F20180311%2Fus-west-2%2Fs3%2Faws4_request&X-Amz-Date=20180311T163814Z&X-Amz-Expires=60&X-Amz-SignedHeaders=host&X-Amz-Signature=2d8b5f7f176727aad0651c75a645a515636c2a31803a0e74a52b3b1f8061fd71"
```

Comment:
You can use images from `/app/assets/images` 💃

Reply:
Ohh thanks! So, these are the only changes I need to do in order to approve the PR?

Reply:
@julianguyen I see there are aliases for app folder and config but the path you pointed me at is outside of the client/ folder! How can I access those files? Thanks

Reply:
@julianguyen I added an alias for the app/ folder in webpack base config file and then imported the file on top of the stories index! Then assigned the import to the src props of the Avatar! Is this accepted or we need to do sth else?